src/themes/teams/components/Menu/menuVariables.ts

```diff
@@ -1,52 +1,78 @@
+import { getColorSchemeWithCustomDefaults } from '../../../../lib'
+import { ColorValues, ColorScheme, SiteVariablesPrepared } from '../../../types'
+
+export type MenuColorScheme = Pick<ColorScheme, 'foreground' | 'background' | 'border'>
+
 export interface MenuVariables {
+  colorScheme: ColorValues<MenuColorScheme>
+
   color: string
+  focusedColor: string
+  disabledColor: string
+  primaryActiveColor: string
+  primaryFocusedColor: string
 
-  activeColor: string
+  backgroundColor: string
   activeBackgroundColor: string
   focusedBackgroundColor: string
-  borderColor: string
-
-  iconOnlyActiveColor: string
-  primaryActiveColor: string
   primaryActiveBackgroundColor: string
-  primaryActiveBorderColor: string
-
-  primaryFocusedColor: string
   primaryFocusedBackgroundColor: string
 
+  borderColor: string
   primaryBorderColor: string
-  primaryHoverBorderColor: string
-  primaryUnderlinedBorderColor: string
-
-  disabledColor: string
+  primaryActiveBorderColor: string
 
   lineHeightBase: string
 }
 
-export default (siteVars: any): MenuVariables => {
-  return {
+export default (siteVars: SiteVariablesPrepared): MenuVariables => {
+  const v: MenuVariables = {
     color: siteVars.gray02,
+    focusedColor: siteVars.black,
+    disabledColor: siteVars.gray06,
+    primaryActiveColor: siteVars.white,
```

Comment:
It might make sense to get rid of specific variables like this `primaryActiveColor` and just set `primary.foreground.active` to `siteVars.white` directly.

Reply:
Flat variables are preferred for now for consistency, ease of merging, and ease of discovery.

Once we formalize the structure of our design system, we might have nested values like this. For now, let's keep consistent and flat.

Reply:
Okay, does that mean we should have the flat variable inherit from the nested color scheme values? Or get rid of the color scheme nested values altogether?

Reply:
to be updated after we discuss